nixpkgs_review/nix.py

```diff
@@ -51,6 +51,7 @@ def _nix_eval_filter(json: Dict[str, Any]) -> List[Attr]:
         [
             "tests.nixos-functions.nixos-test",
             "tests.nixos-functions.nixosTest-test",
+            "tests.writers",
```

Comment:
I feel like this still is going to be endless.

Reply:
If not we need to introduce some kind of regex.